ext/drwrap/drwrap.c

```diff
@@ -2345,9 +2348,22 @@ drwrap_after_callee(app_pc retaddr, reg_t xsp)
     }
 }
 
+static void
+drwrap_after_callee_cleanup_only(app_pc retaddr, reg_t xsp)
+{
+    void *drcontext = dr_get_current_drcontext();
+    per_thread_t *pt = (per_thread_t *)drmgr_get_tls_field(drcontext, tls_idx);
+    /* To reduce overhead for non-drbbdup cases we avoid an extra arg to a common
```

Comment:
Wondering why we didn't add the arg, it's just `drwrap_after_callee_func` and `drwrap_after_callee` right?

Reply:
Overhead is important for these calls; would prefer to not add anything to the common case if we can help it.